src/celeritas/ext/GeantSetup.cc

```diff
@@ -138,6 +138,8 @@ GeantSetup::GeantSetup(std::string const& gdml_filename, Options options)
         run_manager_->SetUserInitialization(detector.release());
 
         // Construct the physics
+        CELER_LOG(status) << "Initializing Geant4 phyics";
+        ScopedTimeLog scoped_time;
```

Comment:
I don't think it really matters since the end of scope is what counts most (i.e. it prints there) and the log statement takes effectively zero time.  I think it makes more sense to put it below the status? I think this particular instance should be removed since constructing the physics list is so cheap (it's building the tables that can be expensive).